english/coffee/coffee.md

```diff
@@ -16,6 +16,6 @@ After having your coffee do you want to:
 
 [Bake yourself something sweet to go with this delicious brew?](cookie/bake-cookies.md)
 
-
+[Drink cappuccino?](drink-cappuccino/drink-cappuccino.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.